src/plugins/intel_cpu/src/utils/jit_kernel.cpp

```diff
@@ -15,6 +18,12 @@ namespace {
 template<typename RegType>
 using registers = std::array<std::reference_wrapper<const RegType>, 16>;
 
+bool isRegAllocable(int id) {
+    return id != abi_param1.getIdx()          // function argument
+            && id != Operand::Code::RSP       // stack pointer
+            && id != Operand::Code::RBP;      // frame pointer
```

Comment:
1. I guess RBP is wifely used in other JIT kernels for computations. Why we protect it here?
2. JIT kernel might use abi_param1 to pass call arguments. How we preserve this register?

Reply:
1. The RBP removed from this expression.
2. I think abi_param1 register should not be changed during code generation. This is useful for restoring the original input values passed to the jit kernel.